src/pages/configurePatch/ConfigurePatchCore.tsx

```diff
@@ -52,7 +52,7 @@ export const ConfigurePatchCore: React.FC<Props> = ({ patch }) => {
   const { tab: urlTab } = useParams<{ tab: PatchTab | null }>();
 
   const { project, variantsTasks, id } = patch;
-  const { variants, tasks } = project;
+  const { variants } = project;
```

Comment:
We should remove the tasks field from the query that fetches this data. If we deploy this pr first with those changes, We wont risk breaking anything on the backend deploy.